master/buildbot/reporters/base.py

```diff
@@ -81,6 +88,12 @@ def _does_generator_want_key(self, generator, key):
 
     @defer.inlineCallbacks
     def _got_event(self, key, msg):
+        while self._event_queue:
+            yield self._event_queue.popleft()
+
+        event = defer.Deferred()
```

Comment:
Please rename `_event_queue` to `_pending_got_event_calls` and `event` to just `d`.